sunpy/net/helio/tests/test_helio.py

```diff
@@ -294,6 +294,14 @@ def test_client_search(client):
     assert len(res) == 10
 
 
+def test_client_search_limit(client):
```

Comment:
This test can be reduced to just testing the attr directly instead of doing a search.